.github/workflows/build.yml

```diff
@@ -0,0 +1,71 @@
+name: Build
```

Comment:
We probably need to remove this file in our fork. Github is blocking the merge because of the workflow build failures.